docs/development/releasing.rst

```diff
@@ -432,59 +434,51 @@ release. The rest of the procedure is the same as any other release as
-The script is not perfect and still needs a little work, but it will get most of
-the work done.  For example, if
-the current bug fix branch is called 'v1.2.x' run it like so::
-
-    $ suggest_backports.py astropy astropy v1.2.x -f backport.sh
-
-This will search GitHub for all issues assigned to the next bug fix release
-milestone that's associated with the given bug fix branch ('v1.2.2' for
-example), find the commits that fix those issues, and will generate a shell
-script called ``backport.sh`` containing all the ``git cherry-pick`` commands
-to backport all those fixes.
-
-The ``suggest_backports.py`` script will typically take a couple minutes to
-run, but once it's done simply execute the generated script from within your
-local clone of the Astropy repository::
+To aid this process, there are a series of related scripts in the
+`astropy-tools repository`_, in the ``pr_consistency`` directory.  These scripts
+essentially check that the above two conditions are met. Detailed documentation
+for these scripts is given in their repository, but here we summarize the basic
+workflow.  Run the scripts in order (they are numbered 1.<something>.py,
+2.<something>.py, etc.), entering your github login credentials as needed (if
+you are going to run them multiple times, using a ``~/.netrc`` file is
+recommended). The script to actually scheck consistency should be run as:
 
-    $ ./backport.sh
+    $ python 4.check_consistency.py > consistency.html
 
-This will checkout the appropriate bug fix branch ('v1.2.x' in this example),
-do a ``git pull upstream v1.2.x`` to make sure it's up to date, and then start
-doing cherry-picks into the bug fix branch.
+Which will generate a simple web page that shows all of the areas where either
+a pull request was merged into master but is *not* in the relevant release that
+it has been milestoned for, as well as any changelog irregularities (i.e., PRs
+that are in the wrong section for what the github milestone indicates).  You'll
+want to correct those irregularities *first* before starting the backport
+process (re-running the scripts in order as needed).
 
-.. note::
+The end of the ``consistency.html`` page will then show a series of
+``git cherry-pick`` commands to update the maintainence branch with the PRs that
+are needed to make the milestones and branches consistent.  Make sure you're in
+the correct maintainence branch with e.g.,
 
-    As discussed earlier, cherry-pick may result in merge conflicts.  If this
-    occurs, the ``backport.sh`` script will exit and the conflict should be
-    resolved manually, followed by running ``git commit``.  To resume the
-    ``backport.sh`` script after the merge conflict, it is currently necessary
-    to edit the script to either remove or comment out the ``git cherry-pick``
-    commands that already ran successfully.
+    $ git checkout v1.3.x
```

Comment:
At this point, double check that local v1.3.x branch is at the top of the remote v1.3.x branch.